package.json

```diff
@@ -77,9 +77,9 @@
         "publish_package": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run publish_package $2 ;}; f",
         "serve": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run serve $npm_config_open;}; f",
         "start": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run start ;}; f",
-        "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && npm run test:eslint-all && JEST_MAX_WORKERS=6 npm run test:jest  ;}; f",
+        "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && npm run test:eslint-all && npm run test:jest  ;}; f",
         "test:stylelint": "stylelint \"./packages/*/src/**/*.s(a|c)ss\"",
-        "test:jest": "jest --all --maxWorkers=${JEST_MAX_WORKERS:-'45%'}",
```

Comment:
dont need the max workers anymore as by default `jest` uses 50 percent of the cpu cores.

Reply:
Shouldn't we use 100% of CPU on github actions to make the tests run faster? It's the only thing the test workflow will be running right?

Reply:
its possible that the runner crashes the jest recommendation is to use 50 percent of cpu